SuperBuild/External_ITK.cmake

```diff
@@ -79,6 +93,20 @@ if(NOT DEFINED ITK_DIR AND NOT Slicer_USE_SYSTEM_${proj})
   set(EP_SOURCE_DIR ${CMAKE_BINARY_DIR}/${proj})
   set(EP_BINARY_DIR ${CMAKE_BINARY_DIR}/${proj}-build)
 
+  if( ${Slicer_ITK_VERSION_MAJOR} GREATER_EQUAL 5)
+    list(APPEND EXTERNAL_PROJECT_OPTIONAL_CMAKE_CACHE_ARGS
+      -DITK_LEGACY_REMOVE:BOOL=OFF   #<-- Allow LEGACY ITKv4 features for now.
+      -DITK_LEGACY_SILENT:BOOL=ON    #<-- Silence for initial ITKv5 migrations.
+      -DModule_ITKDeprecated:BOOL=ON #<-- Needed for ITKv5 now. (ITK MultiThreader backwards compatibility.)
```

Comment:
`ITKDeprecated` is mostly for MutexLock -> comment needs an update

Reply:
@dzenanz #include <itkMultiThreader.h> was where I identified that this was needed, and that is why the comment is as it is.  

johnsonhj@Hanss-MBP$ find . -name "itkMultiThreader.h"
./Modules/Compatibility/Deprecated/include/itkMultiThreader.h